string name
uint8 color   # 0=Red, 1=Yellow, 2=Green
float32 xmin
float32 xmax
float32 ymin
float32 ymax
float32 stopline_x
float32 stopline_y


